src/app/cli/default_cli_delegate.cpp

```diff
@@ -63,8 +63,7 @@ void DefaultCliDelegate::afterOpenFile(const CliOpenFile& cof)
     return;
 
   if (cof.listLayers) {
-    for (doc::Layer* layer : cof.document->sprite()->allVisibleLayers())
-      std::cout << layer->name() << "\n";
+    std::cout << cof.document->sprite()->visibleLayerHierarchyAsString() << "\n";
```

Comment:
Why are we not creating a new command? I think unnecessary. If the user wants to see _only_ layers, it can check with last `/` character

Reply:
I did a search in GitHub and there are places where changing the --list-layers behavior can break existing scripts (e.g. https://github.com/ZenCoderGames/ZRoguelikeGame/blob/6b21228a98c33bf8a5ef8251e564f85c226ebc66/addons/AsepriteWizard/aseprite/aseprite.gd#L118)

I think that for any change to this behavior we should add a new parameter.